### PR TITLE
Setup tests

### PR DESCRIPTION
**Pull request by goranalkovic-infinum**
This took way more than I hoped it would, ran into some weird issues and path brainf##ks, but all is good now 😄 

Changelog:
- `AbstractCli` wasn't detecting the test environment correctly, added the condition that makes it work now
- extracted command name to a const in `SetupCli` and `UpdateCli`
- added check in `UpdateCli` to detect if we're in a test environment and make it point to `setup.json` in `cliOutput` instead of libs root folder
- added tests for `SetupCli` and `UpdateCli`

### CODE REVIEW

File: tests/Setup/SetupCliTest.php [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	$wpCliMock
+		->shouldReceive('success')
+		->andReturnArg(0);
+
+	$wpCliMock
+		->shouldReceive('error')
+		->andReturnArg(0);
+
+	$this->setup = new SetupCli('boilerplate');
+});
```
Comment (dingo-d) [inline tests/Setup/SetupCliTest.php:24]: There seem to be some issues with the indentation here.

File: tests/Setup/UpdateCliTest.php [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+	$wpCliMock
+		->shouldReceive('runcommand')
+		->andReturnArg(0);
+
+	$wpCliMock
+		->shouldReceive('log')
+		->andReturnArg(0);
+
+	$this->update = new UpdateCli('boilerplate');
+});
```
Comment (dingo-d) [inline tests/Setup/UpdateCliTest.php:34]: Indentation

File: tests/Setup/UpdateCliTest.php [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+	// ... so you can copy the example setup.json inside
+	copy(dirname(__FILE__, 3) . '/src/Setup/setup.json', dirname(__FILE__, 3) . '/cliOutput/setup.json');
+
+	// Check if an exception occured
+	$exceptionOccured = false;
+
+	$update = $this->update;
+	$update([], []);
+});
```
Comment (dingo-d) [inline tests/Setup/UpdateCliTest.php:61]: What is being tested here? I think you wanted to test something like

```php
$this->assertFalse($exceptionOccured);
```

Right? If an exception occurred, that variable would be true, and the assertion would be false.

**Review by dingo-d — CHANGES_REQUESTED** [review]
Left one comment and some indentation issues. Great job 👍🏼

File: tests/Setup/UpdateCliTest.php (hunk 3 above)
Comment (goranalkovic-infinum) [reply tests/Setup/UpdateCliTest.php]: I was trying something out and forgot to do the couple of undos to get the try/catch and the assert back. Fixed now 😄

**Review by dingo-d — APPROVED** [review]

**Review by iruzevic — APPROVED** [review]

**Pull request merged**